test/osb_and_plugin_test/osb_test/catalog_test.go

```diff
@@ -92,6 +92,17 @@ var _ = Describe("Catalog", func() {
 				resp.Path("$.services[*].dashboard_client[*]").Array().Contains("id", "secret", "redirect_uri")
 				resp.Path("$.services[*].plans[*].random_extension").Array().Contains("random_extension")
 			})
+
+			It("should return the SM offering id in the catalog metadata", func() {
+				resp := ctx.SMWithBasic.GET(smUrlToSimpleBrokerCatalogBroker+"/v2/catalog").WithHeader(brokerAPIVersionHeaderKey, brokerAPIVersionHeaderValue).
+					Expect().
+					Status(http.StatusOK).JSON()
+
+				metadata := resp.Path("$.services[*].metadata").Array().First().Raw()
+				metadataMap, ok := metadata.(map[string]interface{})
+				Expect(ok).To(BeTrue())
+				Expect(metadataMap["sm_offering_id"]).ToNot(BeEmpty())
```

Comment:
dont you want to verify it is the correct ID?

Reply:
agree, validation added